build.sh

```diff
@@ -62,6 +84,13 @@ for PKG_PATH in $(catkin_topological_order --only-folders); do
   test -f "$PKG_PATH/COLCON_IGNORE" && echo "Skipped" && continue
   (
   cd "$PKG_PATH"
+  test -f debian/changelog && GENERATED_DEBIAN_PACKAGE=$(head -n1 debian/changelog | cut -d' ' -f1)
```

Comment:
`debian/changelog` does not exist at this point. Feel free to revert to my proposed change. For reference: https://github.com/jspricke/ros-deb-builder-action/commit/4c058db5d37e26fadff10e66f58e48257c0aa867#diff-4d2a8eefdf2a9783512a35da4dc7676a66404b6f3826a8af9aad038722da6823R83

Reply:
It should if it's been previously generated, right? And if not, that means we'll have to generate the package, wouldn't it? I prefer using `head` instead of `catkin_topological_order`, as it should be quicker.

Reply:
that's a bold assumption, I would not follow that.

Reply:
Well all of it. I assume that the user can do anything between two runs, including deleting `debian/changelog` and I assume their computer is fast enough that the different commands do not matter.

Reply:
That's an interesting case, I'm sure the user can do plenty of other things between two runs that may conflict with this script, but I'm not stubborn enough that this change bothers me that much, I'll update it. We could be arguing ad nauseam, which would be a waste of time.